dev/io.openliberty.jakarta.expressionLanguage.5.0/bnd.bnd

```diff
@@ -29,7 +29,8 @@ Export-Package: \
        uses:="jakarta.el"
 
 Include-Resource: \
-    jakarta=resources/jakarta
+    jakarta=resources/jakarta, \
+    @${repo;org.apache.tomcat:tomcat-el-api;10.1.2;EXACT}!/!(META-INF/maven/*|module-info.class)
```

Comment:
Same flip question about overlay.

Reply:
Same answer - Does not appear to be a problem but probably doesn't hurt to flip it anyway.